docs/scos/dev/front-end-development/202307.0/oryx/technology.md

```diff
@@ -33,7 +33,7 @@ HTML is a markup language used to create web pages and applications. In Oryx, HT
 
 CSS is a style sheet language used to describe the visual presentation of a document written in HTML.
 
-Oryx uses standard CSS syntax and does not use CSS preprocessors because Oryx does not use a global CSS utility library; instead, we have isolated styles per component and because of that we do not want to load global css utils in each component.
+Oryx uses standard CSS syntax. Because Oryx doesn't use a global CSS utility library, it doesn't use CSS preprocessors; instead, styles are isolated per component and global CSS utils are not loaded in them.
```

Comment:
`Oryx uses standard CSS syntax` and `Oryx doesn't use CSS preprocessors` statements logically should be together. So what if we rephrase it:

> Oryx uses standard CSS syntax and does not use CSS preprocessors. Also Oryx doesn't use a global CSS utility library; instead, styles are isolated per component and global CSS utils are not loaded in them.